default.nix

```diff
@@ -8,7 +8,7 @@
   meson,
   ninja,
   nix,
-  nixpkgs-fmt,
+  nixfmt-rfc-style,
```

Comment:
this dep seems to be a testing quirk that we can remove it. I will push your branch to do that.